packages/video_player/video_player_android/CHANGELOG.md

```diff
@@ -284,3 +296,4 @@
 ## 2.2.17
 
 * Splits from `video_player` as a federated implementation.
+  `
```

Comment:
Please undo this change too.

Also nit: the rest of the changes seem like auto-formatter changes, that are not needed.